wfe/web-front-end.go

```diff
@@ -884,6 +884,14 @@ func (wfe *WebFrontEndImpl) Challenge(
 		notFound()
 		return
 	}
+
+	// After expiring, challenges are inaccessible
+	if authz.Expires == nil || authz.Expires.Before(time.Now()) {
```

Comment:
Looks like we haven't yet added a clock element to WFE. Could you please add one, and change this to use it? We'll also want a clock element on `mocks.StorageAuthority`, to be initialized by test code to match other necessary clocks in use. That will allow us to write tests that don't depend on `time.Now()`.